Modules/HR/Http/Controllers/EmployeeController.php

```diff
@@ -28,10 +28,25 @@ public function __construct(EmployeeService $service)
      */
     public function index(Request $request)
     {
+        $this->authorize('list', Employee::class);
+        $search = request()->query('employeename') ?: '';
+        $employeeData = Employee::with('employees');
         $filters = $request->all();
         $filters = $filters ?: $this->service->defaultFilters();
         $name = request('name');
-        $employeeData = Employee::where('staff_type', $name)->get();
+        $employeeData = Employee::where('staff_type', $name)
+            ->leftJoin('project_team_members', 'employees.user_id', '=', 'project_team_members.team_member_id')
+            ->selectRaw('employees.*, team_member_id, count(team_member_id) as project_count')
+            ->where('project_team_members.ended_on', '!=', 'NULL')
```

Comment:
If the ended_on column is filled then it means that the team member is not a part of project
```suggestion
            ->where('project_team_members.ended_on', '=', 'NULL')
```